Copilot for Xcode/OpenAIView.swift

```diff
@@ -30,63 +30,76 @@ struct OpenAIView: View {
 
                 Form {
                     HStack {
-                        Text("OpenAI API Key")
                         TextField(text: $settings.openAIAPIKey, prompt: Text("sk-*")) {
-                            EmptyView()
-                        }.textFieldStyle(.copilot)
+                            Text("OpenAI API Key")
+                        }.textFieldStyle(.roundedBorder)
                         Button(action: {
                             openURL(apiKeyURL)
                         }) {
                             Image(systemName: "questionmark.circle.fill")
-                        }
-                        .buttonStyle(.plain)
+                        }.buttonStyle(.plain)
                     }
 
                     HStack {
-                        Text("ChatGPT Model")
-                        TextField(text: $settings.chatGPTModel, prompt: Text("gpt-3.5-turbo")) {
-                            EmptyView()
-                        }.textFieldStyle(.copilot)
-
+                        Picker(selection: $settings.chatGPTModel) {
+                            ForEach(ChatGPTModel.allCases, id: \.self) { model in
+                                Text(model.rawValue).tag(model.rawValue)
+                            }
+                        } label: {
+                            Text("ChatGPT Model")
+                        }.pickerStyle(.menu)
                         Button(action: {
                             openURL(modelURL)
                         }) {
                             Image(systemName: "questionmark.circle.fill")
+                        }.buttonStyle(.plain)
+                    }
+                    .onChange(of: settings.chatGPTModel) { newValue in
+                        if let model = ChatGPTModel(rawValue: newValue) {
+                            settings.chatGPTEndpoint = model.endpoint
                         }
-                        .buttonStyle(.plain)
                     }
 
-                    HStack {
-                        Text("ChatGPT Endpoint")
-                        TextField(
-                            text: $settings.chatGPTEndpoint,
-                            prompt: Text("https://api.openai.com/v1/chat/completions")
-                        ) {
-                            EmptyView()
-                        }.textFieldStyle(.copilot)
-                    }
-                    
-                    HStack {
+                    TextField(
+                        text: $settings.chatGPTEndpoint,
+                        prompt: Text("https://api.openai.com/v1/chat/completions")
+                    ) {
+                        Text("ChatGPT Server")
+                    }.textFieldStyle(.roundedBorder)
+
+                    Picker(selection: $settings.chatGPTLanguage) {
+                        ForEach(Locale.availableLocalizedLocales, id: \.self) { localizedLocales in
+                            Text(localizedLocales).tag(localizedLocales)
```

Comment:
When the language is unset, it would be better if we can display some text that the language will be decided by the bot

Reply:
What wording do you consider appropriate?
I have come up with the following candidates

- Auto-detected by ChatGPT
- Recommended by ChatGPT
- ChatGPT's suggested language
- Language selected by ChatGPT
- ChatGPT's preferred language
- Language predicted by ChatGPT

What do you think?

Reply:
Auto-detected by ChatGPT would be fine

Reply:
✅: [1fabffd](https://github.com/intitni/CopilotForXcode/pull/100/commits/1fabffd9d9a90cf2f0a536559bd5b0815f4c0be3)